.github/workflows/integration.yml

```diff
@@ -22,6 +22,19 @@ name: Integration tests
 env:
   IMAGE_NAME: scorecard
   OUTPUT_PATH: output
+
+permissions:
+  actions: none
```

Comment:
note: you need to not mention `name: none`. By default, permission not declared in the list default to `none`.

Reply:
Do you see an issue in having them explicitly calling out as `none`? IMO this specifically encourages the practice of we have explicitly looked into those permissions and have defined them as not needed. This could be an example for other repositories to follow this as a practice. Thoughts?

Reply:
I prefer the condensed version, but I don't have a strong opinion on the matter. Omitting them is more condensed, but it's fine to have them all.

Reply:
thinking about this more. I think the condensed version is easier to read from a human's point of view. The full version makes it hard to see what's not set to `none`. The condensed version feels more like an `allow list`, where only what's needed is declared.

Reply:
Sounds good.